src/nri/fargate_sampler.go

```diff
@@ -0,0 +1,71 @@
+package nri
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/newrelic/infra-integrations-sdk/data/metric"
+	"github.com/newrelic/infra-integrations-sdk/integration"
+
+	"github.com/newrelic/nri-docker/src/raw/aws"
+)
+
+const clusterNameLabel = "com.amazonaws.ecs.cluster"
+
+// FargateSampler is a sampler to get container data from an HTTP endpoint.
+type FargateSampler struct {
+}
+
+// SampleAll will sample all the information this sampler can fetch and populate it in the given integration.
+func (f *FargateSampler) SampleAll(i *integration.Integration) error {
+	httpClient := &http.Client{Timeout: 5 * time.Second}
```

Comment:
will 5 seconds be enough? maybe make it a bit higher, it's not something that will block everything from running